src/Canvas2d/babylon.canvas2d.ts

```diff
@@ -1688,7 +1688,15 @@
                 this._worldSpaceNode = settings.customWorldSpaceNode;
                 this.applyCachedTexture(null, null);
             }
+
+            this.propertyChanged.add((e, st) => {

```

Comment:
This is just because I would like to merge this now but I cannot yet merge ts2 stuff